.github/workflows/build.yml

```diff
@@ -12,24 +12,20 @@ jobs:
         node-version: [8.x, 10.x, 12.x, 14.x, 16.x]
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
     - name: Setting up python 2.X
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: '2.x'
         architecture: 'x64'
-    - name: Installing wheel package for pip # This makes the warning about legacy installing go away.
-      uses: BSFishy/pip-action@v1
-      with:
-        packages: wheel
-    - name: Installing pyxform # Install pyxform dependency.
-      uses: BSFishy/pip-action@v1
-      with:
-        packages: git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic
+    - name: Install wheel
+      run: pip install wheel
```

Comment:
Slightly unrelated, but why do we need `wheel`?  I read through [the documentation](https://pip.pypa.io/en/stable/cli/pip_wheel/) and I am still not sure I understand exactly what it does (but then I don't have too much experience with Python....).  Either way, I don't think I ever installed this on my local machine before using pyxform, so I am not sure why we need it here....

Reply:
No idea. I'll merge this but let me know if you think it's worth raising a new issue to investigate...